server/src/controllers/Auth.controller.js

```diff
@@ -68,4 +69,59 @@ export default class AuthController {
       message: 'User registered successfully'
     });
   }
+
+  /**
+   * @method login
+   *
+   * @param {object} req request
+   * @param {object} res response
+   *
+   * @returns {object} returns status code, data and message properties
```

Comment:
You can remove the returns since it's sort of repetitive i.e
```suggestion
   * @returns {object} status code, data and message properties
```

Reply:
done...